.github/actions/jlab-a11y-test/action.yml

```
@@ -0,0 +1,116 @@
+name: Run accessibility regression tests on JupyterLab
+description: |
+  Builds JupyterLab from source, then runs accessibility regression tests
+  against it using Playwright. Outputs the results to a markdown file which is
+  then copied to the GitHub workflow run summary.
+inputs:
+  repository:
+    description: JupyterLab repository, for example gabalafou/jupyterlab
+  ref:
+    description: ref, branch, or SHA
+runs:
+  using: composite
+  steps:
+    - name: List environment variables
+      run: env
+      shell: bash -l {0}
+
+    - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+      shell: bash
+    - run: echo "🐧 This job is now running on a ${{ runner.os }} server hosted by GitHub!"
+      shell: bash
+    - run: echo "🔎 The name of your branch is ${{ github.ref }} and your repository is ${{ github.repository }}."
+      shell: bash
+    - run: |
+        echo "github.workspace = ${{ github.workspace }}"
+        echo "github.sha = ${{ github.sha }}"
+        echo "github.repository = ${{ github.repository }}"
+        echo "github.server_url = ${{ github.server_url }}"
+      shell: bash
+
+    - name: Cache conda 🧠
+      uses: actions/cache@v2
```

```suggestion
      uses: actions/cache@v3
```

We'll want to update this if we want to stop getting warnings from GitHub about Node 12 being deprecated soon.